biojava dependencies error

Hi, I am getting error on Biojava dependencies below, when building core. 

Biojava site claims that http://biojava.org/wikis/BioJava:GetStarted/

BioJava, as of release 4.0.0 is available through Maven Central. or

You can get the latest version of BioJava3+ from the download page BioJava (v) (requires Java 1.6+).
(link to download is not working)

[ERROR] Failed to execute goal on project core: Could not resolve dependencies for project org.mskcc.cbio:core:jar:1.1.0: The following artifacts could not be resolved: org.biojava:biojava3-core:jar:3.0.7, org.biojava:biojava3-structure:jar:3.0.7: Failure to find org.biojava:biojava3-core:jar:3.0.7 in http://www.biojava.org/download/maven/ was cached in the local repository, resolution will not be reattempted until the update interval of biojava-maven-repo has elapsed or updates are forced

--- a/core/pom.xml
+++ b/core/pom.xml
@@ -11,17 +11,6 @@
   <name>Portal Core</name>
   <description>Core libraries shared among other modules</description>
 
-  <repositories>
-    <repository>
-      <id>biojava-maven-repo</id>
-      <name>BioJava repository</name>
-      <url>http://www.biojava.org/download/maven/</url>
-      <releases>
-        <enabled>true</enabled>
-      </releases>
-    </repository>
-  </repositories>
-
   <dependencies>
     <dependency>
       <groupId>org.mskcc.cbio</groupId>
@@ -163,17 +152,18 @@
       <version>2.3</version>
     </dependency>
 
-    <!-- BioJava3 -->
+    <!-- BioJava 4.x -->
     <dependency>
       <groupId>org.biojava</groupId>
-      <artifactId>biojava3-core</artifactId>
-      <version>3.0.7</version>
+      <artifactId>biojava-core</artifactId>
+      <version>4.2.0</version>
     </dependency>
     <dependency>
       <groupId>org.biojava</groupId>
-      <artifactId>biojava3-structure</artifactId>
-      <version>3.0.7</version>
+      <artifactId>biojava-structure</artifactId>
+      <version>4.2.0</version>
     </dependency>
+
     <!-- Mockito -->
     <dependency>
       <groupId>org.mockito</groupId>

--- a/core/src/main/java/org/mskcc/cbio/portal/scripts/ImportPdbUniprotResidueMappingFromMA.java
+++ b/core/src/main/java/org/mskcc/cbio/portal/scripts/ImportPdbUniprotResidueMappingFromMA.java
@@ -34,11 +34,6 @@ package org.mskcc.cbio.portal.scripts;
 
 import java.io.*;
 import java.util.*;
-import org.biojava.bio.structure.*;
-import org.biojava.bio.structure.align.util.AtomCache;
-import org.biojava.bio.structure.io.FileParsingParameters;
-import org.biojava3.core.sequence.compound.*;
-import org.biojava3.core.sequence.loader.UniprotProxySequenceReader;
 import org.mskcc.cbio.portal.dao.*;
 import org.mskcc.cbio.portal.model.*;
 import org.mskcc.cbio.portal.util.*;

--- a/core/src/main/java/org/mskcc/cbio/portal/scripts/ImportPdbUniprotResidueMappingFromSifts.java
+++ b/core/src/main/java/org/mskcc/cbio/portal/scripts/ImportPdbUniprotResidueMappingFromSifts.java
@@ -34,11 +34,11 @@ package org.mskcc.cbio.portal.scripts;
 
 import java.io.*;
 import java.util.*;
-import org.biojava.bio.structure.*;
-import org.biojava.bio.structure.align.util.AtomCache;
-import org.biojava.bio.structure.io.FileParsingParameters;
-import org.biojava3.core.sequence.compound.*;
-import org.biojava3.core.sequence.loader.UniprotProxySequenceReader;
+import org.biojava.nbio.structure.*;
+import org.biojava.nbio.structure.align.util.AtomCache;
+import org.biojava.nbio.structure.io.FileParsingParameters;
+import org.biojava.nbio.core.sequence.compound.*;
+import org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader;
 import org.mskcc.cbio.portal.dao.*;
 import org.mskcc.cbio.portal.model.*;
 import org.mskcc.cbio.portal.util.*;
@@ -237,7 +237,6 @@ public final class ImportPdbUniprotResidueMappingFromSifts {
     private static AtomCache getAtomCache(String dirCache) {
         AtomCache atomCache = new AtomCache(dirCache, true);
         FileParsingParameters params = new FileParsingParameters();
-        params.setLoadChemCompInfo(false);
         params.setAlignSeqRes(true);
         params.setParseSecStruc(false);
         params.setUpdateRemediatedFiles(false);

--- a/core/src/main/java/org/mskcc/cbio/portal/servlet/PfamSequenceServlet.java
+++ b/core/src/main/java/org/mskcc/cbio/portal/servlet/PfamSequenceServlet.java
@@ -32,9 +32,9 @@
 
 package org.mskcc.cbio.portal.servlet;
 
-import org.biojava3.core.sequence.compound.AminoAcidCompound;
-import org.biojava3.core.sequence.compound.AminoAcidCompoundSet;
-import org.biojava3.core.sequence.loader.UniprotProxySequenceReader;
+import org.biojava.nbio.core.sequence.compound.AminoAcidCompound;
+import org.biojava.nbio.core.sequence.compound.AminoAcidCompoundSet;
+import org.biojava.nbio.core.sequence.loader.UniprotProxySequenceReader;
 import org.json.simple.JSONArray;
 import org.json.simple.JSONObject;
 import org.json.simple.JSONValue;

